pinot-common/src/main/java/org/apache/pinot/pql/parsers/pql2/ast/OutputColumnAstNode.java

```diff
@@ -18,6 +18,7 @@
  */
 package org.apache.pinot.pql.parsers.pql2.ast;
 
+import org.apache.pinot.common.request.AggregationInfo;
```

Comment:
No need to change this file?

Reply:
will do that change in another PR. This PR will help ensure that there is no regression